docs/algorithms/density.rst

```diff
@@ -4,6 +4,13 @@
 Density-based reward modeling
 =============================
 
+Density-based reward modeling is an inverse reinforcement learning (IRL) technique
+that eliminates the need for explicit rewards by assigning higher rewards to states or state-action pairs
```

Comment:
"explicit rewards" do you mean that the need that expert trajectories have rewards in them? I'm confused, isn't this how IRL normally works? (i.e. learning the rewards from trajectories without rewards)

Reply:
No you are right I was the one being confused about the purpose of the density algorithm.